shared/src/simulation/activities/unload-vehicle.ts

```diff
@@ -0,0 +1,39 @@
+import { IsInt, IsUUID, Min } from 'class-validator';
+import { uuid, UUID, uuidValidationOptions } from '../../utils';
+import { IsValue } from '../../utils/validators';
+import type {
+    SimulationActivity,
+    SimulationActivityState,
+} from './simulation-activity';
+
+// in ms (see state.currentTime)
+// TODO: make configurable
+const MAGIC_VALUE_UNLOAD = 5000;
```

Comment:
To make this value configurable, I would move it to the `BehaviorState`. Then the user could attach a behavior with the parameters of their choice to a simulated region and the behavior would supply its actions with the selected value.